pipeline/unified/build-unsigned-release-packages.yaml

```diff
@@ -25,8 +25,8 @@ steps:
 
     - template: ./unified-e2e-publish-results.yaml
 
-    - script: yarn build:unified:all --unified-version=$(Build.BuildNumber) --unified-canary-instrumentation-key=$(UnifiedCanaryInstrumentationKey) --unified-insider-instrumentation-key=$(UnifiedInsiderInstrumentationKey) --unified-prod-instrumentation-key=$(UnifiedProdInstrumentationKey)
-      displayName: yarn build:unified:all
+    - script: npx grunt build-unified-all --unified-version=$(Build.BuildNumber) --unified-canary-instrumentation-key=$(UnifiedCanaryInstrumentationKey) --unified-insider-instrumentation-key=$(UnifiedInsiderInstrumentationKey) --unified-prod-instrumentation-key=$(UnifiedProdInstrumentationKey)
```

Comment:
temporary until external npm issue is resolved https://status.npmjs.org/incidents/lvgmcrfvrvsh